L1Trigger/L1TCaloLayer1/src/HLS_emulator.hh

```diff
@@ -0,0 +1,66 @@
+//The HLS4ML team has provided this interface to compiled model .so objects for the purpose of emulation
+#ifndef HLS4ML_EMULATOR_H_
+#define HLS4ML_EMULATOR_H_
+
+#include <iostream>
+#include <string>
+#include <any>
+#include <dlfcn.h>
+
+class HLS4MLModel {
+public:
+  virtual void prepare_input(std::any input) = 0;
+  virtual void predict() = 0;
+  virtual void read_result(std::any result) = 0;
+  virtual ~HLS4MLModel() = default;
+};
+
+typedef HLS4MLModel* create_model_cls();
+typedef void destroy_model_cls(HLS4MLModel*);
+
+class ModelLoader {
+private:
+  std::string _model_name;
```

Comment:
Underscore should not be used as first character (preferred convention for class members is to have a trailing underscored)